### ISSUE TITLE
Electronics Schematic

### ISSUE THREAD

**Issue by rishchavda** [OpenAgInitiative/gro-hardware#8]
Hi,

I am currently building the Arduino base shield assembly. In your instructions I cannot find any information on how the "S" grove connectors connect to the Arduino. The electronic schematic and motherboard assembly (section 8) only shows how the components connect to the perfboard base shield but not which specific analogue/digital connectors the soldered "S" connectors are then wired to. I may be looking in the wrong place but can you help me?

Thank you,

Rishi

**Comment by AdrianLLu** [comment]
Hey Rishi,
You're correct in this being hard to follow. The FC community has a slack channel where we're discussing the documentation for just this right now. I'll see about getting you on there.
Cheers,
Adrian

**Comment by rishchavda** [comment]
Hi Adrian
Thanks for the fast response! That would be great, I would be happy to help with anything else if you need. I am a Motorsport engineer who happens to find this sort of thing really interesting. My skills are primarily component and mechanism design but am an all round handy man.
Cheers,
Rishi

**Comment by rklosinski** [comment]
Rishi,

You should have received an invitation via email a few hours ago. Thanks
for joining.

Regards,
Rich

**Comment by sp4ghet** [comment]
I'm also interested in this slack community. I'd like to begin building a food computer after I move next week.

Thanks!

**Comment by rklosinski** [comment]
Go to www.plantcubed.com -> Support and complete the registration and I'll
set you up.

Regards,
Rich

**Comment by sp4ghet** [comment]
Cool,

thanks!

**Comment by loftypavement** [comment]
Hi Guys,

I'm having the same issue as well, I've just applied for the slack group, hopefully talk soon!

Cheers

**Comment by activescott** [comment]
+1. I'm just getting ready to solder here and running into the same question: How do the grove connectors connect to the Arduino

**Comment by AdrianLLu** [comment]
I posted about this on the forum
http://forum.openag.media.mit.edu/t/protoshield-wiring-schematic/65

**Comment by activescott** [comment]
Thanks @AdrianLLu . [Your blog post on the protoshield](http://adrianllu.com/2016/04/17/motherboard-build-notes-part-1-micro-controller-protoshield/) with pictures I just found also made the schematic easier to understand too. Thanks for that!

**Comment by sp4ghet** [comment]
I think we can close this issue since there are links to both AdrianLLu's schematic and another wiring diagram from the forums linked in the README.
